examples/tuners/customized_tuner/README.md

```diff
@@ -0,0 +1,50 @@
+# How to install this customized tuner as a builtin tuner
```

Comment:
better to put this file in doc directory, otherwise, it may not be easy to maintain

Reply:
you can keep this readme file, and reference to the corresponding doc page

Reply:
There is a link to this README from document docs/en_US/Tutorial/InstallCustomizedAlgos.md
.

Reply:
yes, still suggest to put this file under docs/. because in this way docs are easier to be maintained, also it is easy to manage Chinese translation.

Reply:
Thanks, moved to docs and keep a reference to docs.